docs/home/whats-new-2-16.md

```diff
@@ -10,7 +10,7 @@ date: 2023-Dec-4
 
 ### Researcher
 
-* <!--  DONE RUN-12597/RUN-12601	TW - Hide IDEs behind runai authentication -->Added support for secure and private access to researcher tools such as Jupyter Notebook, and other tool integrations. In order to enable secure and private access, a *Private* toggle has been added to the *Environment* card which enables Authentication and authorization for Workload URLs. This toggle sets a flag of `isPrivate` in the `connections` section of a policy for the connection type `ExternalUrl`. When enabled, this limits the access of this tool to only the creator, or any user in the same scope as the creator. The *Private* toggle appears on any form that requires an *Environment* to be selected. For more information, see [Creating a new Workspace](../Researcher/user-interface/workspaces/create/workspace-v2.md).
+* <!--  DONE RUN-12597/RUN-12601	TW - Hide IDEs behind runai authentication -->Added enterprise level security for researcher tools such as Jupyter Notebooks, VSCode, or any other URL associated with the workload. Using this feature, anyone within the organization requesting access to a specific URL will be redirected to the login page to be authenticated and authorized. This results in protected URLs which cannot be reached from outside the organization. Researchers can enhance the URL privacy by using the *Private* toggle which means that only the researcher who created thw workload can is authorized to access it. The *Private* toggle is available on workloads where an environment with an external URL configured for it. The toggle is set per tool. This toggle sets a flag of `isPrivate` in the `connections` section of a policy for the connection type `ExternalUrl`. For more information, see [Creating a new Workspace](../Researcher/user-interface/workspaces/create/workspace-v2.md).
```

Comment:
some typos/fixes/changes

1. researcher who created thw workload can is authorized to access it -->  researcher who created the workload is authorized to access it
2. The *Private* toggle is available on workloads where an environment with an external URL configured for it --> The *Private* toggle is available per tool that uses an external URL as a connection type  and are located in the workload creation from in the UI in the environment section.
3. The toggle is set per tool --> delete this (repetitive)